src/js/TuningSliders.js

```diff
@@ -0,0 +1,313 @@
+    // and the sliders are shown, otherwise sliders are grayed out and centered
+    this.pidSlidersUnavailable = false;
+    let currentPIDs = [];
+    PID_names.forEach(function(elementPid, indexPid) {
+        let searchRow = $('.pid_tuning .' + elementPid + ' input');
+        searchRow.each(function (indexInput) {
+            if (indexPid < 3 && indexInput < 5) {
+                currentPIDs.push($(this).val());
+            }
+        });
+    });
+    this.calculateNewPids();
+    PID_names.forEach(function(elementPid, indexPid) {
+        let searchRow = $('.pid_tuning .' + elementPid + ' input');
+        searchRow.each(function (indexInput) {
+            if (indexPid < 3 && indexInput < 5) {
+                if (currentPIDs[indexPid * 5 + indexInput] != $(this).val()) {
+                    TuningSliders.pidSlidersUnavailable = true;
+                }
+                $(this).val(currentPIDs[indexPid * 5 + indexInput]);
+            }
+        });
+    });
+    if ($('input[id="useIntegratedYaw"]').is(':checked')) {
+        this.pidSlidersUnavailable = true;
+    }
+    if (this.pidSlidersUnavailable) {
+        $('.tuningPIDSliders').css('opacity', 0.3);
+        $('.tuningPIDSliders output').hide();
+        $('.slidersDisabled').show()
+        $('#tuningMasterSlider, #tuningPDRatioSlider, #tuningPDGainSlider, #tuningResponseSlider').val(1);
+    } else {
+        $('.tuningPIDSliders').css('opacity', '');
+        $('.tuningPIDSliders output').show();
+        $('.slidersDisabled').hide();
+        this.initPidSlidersPosition();
+    }
+    if ((PIDs[1][0] > 110 || PIDs[1][1] > 120 || PIDs[1][2] > 70 || ADVANCED_TUNING.dMinPitch > 40) && !this.pidSlidersUnavailable) {
+        $('.sliderHighWarning').show();
+    } else {
+        $('.sliderHighWarning').hide();
+    }
+}
+
+TuningSliders.updateFilterSlidersDisplay = function() {
+    // check if filters changed manually by comapring current value and those based on slider position
+    // if equal filter slider is shown, otherwise it is grayed out and centered
+    this.filterGyroSliderUnavailable = false;
+    this.filterDTermSliderUnavailable = false;
+
+    if ($('.pid_filter input[name="gyroLowpassDynMinFrequency"]').val() != Math.round(this.FILTER_DEFAULT.gyro_lowpass_dyn_min_hz * this.gyroFilterSliderValue) ||
+        $('.pid_filter input[name="gyroLowpassDynMaxFrequency"]').val() != Math.round(this.FILTER_DEFAULT.gyro_lowpass_dyn_max_hz * this.gyroFilterSliderValue) ||
+        $('.pid_filter select[name="gyroLowpassDynType"]').val() != this.FILTER_DEFAULT.gyro_lowpass_type ||
+        $('.pid_filter input[name="gyroLowpass2Frequency"]').val() != Math.round(this.FILTER_DEFAULT.gyro_lowpass2_hz * this.gyroFilterSliderValue) ||
+        $('.pid_filter select[name="gyroLowpass2Type"]').val() != this.FILTER_DEFAULT.gyro_lowpass2_type) {
+
+        $('.tuningFilterSliders .sliderLabels tr:first-child').css('opacity', 0.3);
```

Comment:
Maybe is better to add a class dinamically with the CSS content? Adding CSS content in the JS part makes difficult to make changes to the style later.

Reply:
Probably this will be refactored with the new button to enable sliders, so there won't be any need for this.